v2/index.html

```diff
@@ -1160,32 +1160,36 @@ <h4>Chords</h4>
         </section>
         <section>
             <h4>Beaming</h4>
+            <div class="issue" data-number="24"></div>
             <div class="issue" data-number="45"></div>
-            <table class="simple">
-                <thead>
-                <tr>
-                    <th>Code</th>
-                    <th>Remarks</th>
-                </tr>
-                </thead>
-                <tbody>
-                <tr>
-                    <td><code>{</code></td>
-                    <td>beginning of beaming</td>
-                </tr>
-                <tr>
-                    <td><code>}</code></td>
-                    <td>end of beaming</td>
-                </tr>
-                </tbody>
-            </table>
-
+            <div class="issue" data-number="84"></div>
+            <p>
+                Beamed notes are encoded using braces <code>{}</code>.
+            </p>
+            <p>
+                Beam groups MUST start with an opening brace <code>{</code>, and
+                MUST end with a closing brace, <code>}</code>. Nested beam groups
+                MUST NOT occur.
+            </p>
+            <p>
+                Notes with duration values of <code>0</code>, <code>1</code>, <code>2</code>,
+                <code>4</code>, and <code>9</code> MUST NOT occur within
+                a beam group. Notes with other duration values MAY occur within
+                a beam group.
+            </p>
+            <p>
+                Rests SHOULD NOT occur within a beam group.
+            </p>
+            <p>
+                Beam groups MUST NOT occur in Mensural encodings.
```

Comment:
Agree. Another think to check in our data.

Reply:
Maybe it should be a `SHOULD`. It is not impossible to find some sources in proper mensural notation with *fusae* beamed together. I am thinking keyboard music from the 1600.